.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
           cache: 'yarn'
       - name: Fetch Node Packages
         run: |
-          yarn --frozen-lockfile
+          yarn --immutable
```

Comment:
Note : --frozen-lockfile doesn't exist anymore and was changes to --immutable with yarn 2